openscap.spec

```diff
@@ -0,0 +1,205 @@
+# This spec file is not synchronized to the Fedora downstream.
+# It serves as Fedora CI configuration and as support for downstream updates.
+Name:           openscap
+Version:        1.3.4
+Release:        4%{?dist}
```

Comment:
```suggestion
Release:        0%{?dist}
```

Reply:
I wonder whether it's not worth to monkey-patch the version number according to the last tag in that branch or something like that.